test/regression/sql/simple_functions.sql

```diff
@@ -10,3 +10,11 @@ SELECT test_escape_uri('https://duckdb.org');
 SELECT test_escape_uri('https://duckdb.org/with space');
 
 SELECT test_escape_uri('foo $ bar # baz / qux');
+
+SELECT test_escape_uri('foo 😀 bar # baz / qux');
+
+SELECT test_escape_uri('Hannes Mühleisen');
+
+SELECT test_escape_uri('Hannes M□hleisen');
```

Comment:
What character is this square? I only see a square. Maybe add a comment about what it is.

Reply:
This is a tribute to the man, he'll know why :-) 
cc @hannes